auth/src/main/java/org/aerogear/mobile/auth/configuration/DefinedBrowserType.java

```diff
@@ -0,0 +1,19 @@
+package org.aerogear.mobile.auth.configuration;
+
+/**
+ * Represents the different browser options available.
+ */
+public enum DefinedBrowserType {
+    /**
+     * This represents a Google Chrome browser where the custom tab and versions can be specified.
+     */
+    CHROME(),
```

Comment:
I don't think the parentheses should be here.

Reply:
Left that in too, thanks 👍